### DISCUSSION TITLE
Transfer transaction with delayed end date

### DISCUSSION THREAD

**Discussion by Anton-V-K** [moneymanagerex/moneymanagerex#3930] [Q&A]
If I transfer money from my account in one bank to another my account in another bank, this operation may be performed with a delay (like next day or later).
How can I enter such transaction into MMEX?
I don't see a second date in the New Transaction dialog even if I select the "Advanced" mode.

**Comment by TomAff** [comment]
I added a category "Money in Transit". 
Date 1 Cash out 
Date 2 Cash in

Category should be 0 after the transfer.

**Reply by Anton-V-K** [reply]
Are you using the extra category "Money in Transit" instead of the built-in category "Transfer" for all transfer transactions?
Why then not using an extra account (probably one per currency)?

**Reply by TomAff** [reply]
Correct.
On account 1
01.01.2021 widthdrawal 100 => category Money in Transit

On Account 2
02.01.2021 deposit 100 => category Money in Transit

I only have one currency, so for me it's the easiest way.
Creating a separate account for transits can be a possibility too.
But not for me.
I export all data to csv and import it into excel. In the csv I only have withdrawal and deposit.
If I use transfer I have to check which account is + which is -

**Reply by vomikan** [reply]
Using this logic:

![image](https://user-images.githubusercontent.com/6836805/138551603-1602ae98-28eb-4bfd-bc19-794b6faa6ebf.png)

![image](https://user-images.githubusercontent.com/6836805/138551605-7e18b425-8b86-4ffa-9b21-4c374ca74259.png)

**Comment by vomikan** [comment]
There was an attempt to implement similar functionality. It was possible to change the transaction statuses for one and the other accounts, but without specifying the second date.
https://github.com/moneymanagerex/moneymanagerex/pull/1208
